test/e2e/service.go

```diff
@@ -266,10 +270,6 @@ var _ = Describe("Services", func() {
 				CreateExternalLoadBalancer: true,
 			},
 		}
-
-		By("cleaning up previous service " + serviceName + " from namespace " + ns)
```

Comment:
It doesn't really achieve much in the normal Jenkins case, as in the new K8S cluster, the old service doesn't exist anyway.  It's only useful if you run the same test twice against the same Kubernetes cluster instance, which almost never happens.  And I'm about to but this test in its own namespace anyway, so even that case won't work soon.